### PR TITLE
Feature/typeit

### PR DESCRIPTION
**Pull request by rshiva**

### CODE REVIEW

File: app/javascript/packs/components/header.tsx [hunk 1]
```diff
@@ -4,14 +4,14 @@ import { HeroBanner } from "./hero_banner";
 
 interface Props {
   page: string;
-  title: string;
+  // title: string;
```
Comment (saurabhbhatia) [inline app/javascript/packs/components/header.tsx]: why is this removed ? it's being used in 2 other pages - contact and reveal post page. it will break those pages.

File: app/javascript/packs/components/hero_banner.tsx [hunk 2]
```diff
@@ -35,7 +39,14 @@ export const HeroBanner: React.FC<Props> = (props: Props) => (
   <div className="flex flex-column w-full border-b-1 bg-maize-crayola">
     <div className="w-1/2 md:w-full md:py-4 sm:w-full sm:py-4 flex items-center text-center justify-center">
       <h1 className="text-5xl font-heading">
-        {props.title}
+        Share
```
Comment (saurabhbhatia) [inline app/javascript/packs/components/hero_banner.tsx]: can we have it only on home page ? it's not relevant to other pages. You can have a conditional here. Some pseudocode is as below -

```
{(props.page === "home") ? {<Typed ...>} : {props.title}}
```

File: app/javascript/packs/components/hero_banner.tsx [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
 import { ContactUs } from "./icons/contact_us";
 import { HashLink as Link } from 'react-router-hash-link';
 import { Unlock } from "./icons/unlock";
+import Typed from 'react-typed';
+
 
 interface Props {
   page: string;
-  title: string;
+  // title: string;
```
Comment (saurabhbhatia) [inline app/javascript/packs/components/hero_banner.tsx]: you can also make this optional with

```
title?: string
```

instead of fully removing it.

**Review by saurabhbhatia — COMMENTED** [review]
good stuff, make title optional instead of removing it completely

File: app/javascript/packs/components/hero_banner.tsx (hunk 2 above)
Comment (rshiva) [reply app/javascript/packs/components/hero_banner.tsx]: I know little react and no typescript so I have no idea what you are talking about. .. trying my best 😥

Comment (saurabhbhatia) [reply app/javascript/packs/components/hero_banner.tsx]: https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator

Comment (saurabhbhatia) [reply app/javascript/packs/components/hero_banner.tsx]: Something like this 😄

Comment (saurabhbhatia) [reply app/javascript/packs/components/hero_banner.tsx]: Updated the link above
